app/gui/src/dashboard/components/dashboard/columnHeading/ModifiedColumnHeading.tsx

```diff
@@ -56,24 +56,25 @@ export default function ModifiedColumnHeading(props: AssetColumnHeadingProps) {
         onPress={hideThisColumn}
       />
       <Button
+        fullWidth
         size="custom"
         variant="custom"
-        className="flex grow justify-start gap-icon-with-text"
+        weight="bold"
+        addonEnd={
+          <img
```

Comment:
Why do we use `img` here?

Reply:
this was how it was before - but good point, will fix soon:tm: